processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        url = self.start_urls[0] + "{i}/0/0/0/0/"
+        last_page = int(response.xpath("//select/option/text()")[-1].extract())
```

Comment:
Is there a class or id that can narrow your search space and make you find this last page in a similar way to the others xpaths you are using? For the same reasons I gave before, if you are not sure that if they can introduce a new unrelated select element and brake this, it might be better trying to think another way of finding this element.

This xpath is very simple and that is very good, but if wasn't looking for every select element in the page it could avoid some problems.

Reply:
`select` with `id="select"` is the one that we need to get. This is enough to get only the one that we are interested.